libs/platform/src/lib/components/form/form-group/form-group.component.ts

```diff
@@ -203,19 +205,31 @@ export class FormGroupComponent implements FormGroupContainer, OnInit, AfterCont
         this._useForm = coerceBooleanProperty(value);
     }
 
+    /**
+     * specify the column layout in the format `XLn-Ln-Mn-Sn` where n is the number of columns and can be different for each size.
+     * eg: XL2-L2-M2-S1 would create 2-column layouts for XL, L, and M sizes and single-column layout for S size.
+     */
+    @Input()
+    columnLayoutType: string;
+
     @ContentChild('i18n', { static: true })
     i18Template: TemplateRef<any>;
 
     @Output()
     onSubmit: EventEmitter<any> = new EventEmitter<any>();
 
+    @ViewChildren(FormFieldGroupComponent) formFieldGroup;
```

Comment:
Can you hide this from api files? - by adding `/** @hidden */` comment.

Let's also add type

Reply:
Unnecessary property. Removed